package.json

```diff
@@ -102,4 +101,4 @@
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
   }
-}
+}
```

Comment:
I guess your editor should automate the newline at the end of files to be consistent with our codebase.

Reply:
resolved